Examples/IcmpFileTransfer/IcmpFileTransfer-pitcher.cpp

```diff
@@ -36,12 +36,13 @@ void usleep(__int64 usec)
 	LARGE_INTEGER ft;
 
 	ft.QuadPart = -(10 * usec);  // Convert to 100 nanosecond interval, negative value indicates relative time
-
+	                             // NULL is Used for WindowAPIs and no need to change to nullptr for C++11
 	timer = CreateWaitableTimer(NULL, TRUE, NULL);
 	if (timer == nullptr)
 	{
 		throw std::runtime_error("Could not create waitable timer with error: " + std::to_string(GetLastError()));
 	}
+	// NULL is Used for WindowAPIs and no need to change to nullptr for C++11
```

Comment:
What about this? Maybe it is clearer with the reference link

```suggestion
	// NULL is used instead of nullptr for Windows APIs. Check https://devblogs.microsoft.com/oldnewthing/20180307-00/?p=98175
```

If it is ok, please change all comments with this one and thanks for contribution 🙏